### PR TITLE
add space name to proposal information section

### PR DESCRIPTION
**Pull request by rootial**
## Summary
Fixes  https://github.com/snapshot-labs/snapshot/issues/1226

## Test
Kapture 2021-12-11 at 22 35 12

## Reviewer
@bonustrack @samuveth @ChaituVR

### CODE REVIEW

**Review by mktcode — APPROVED** [review]

**Comment by rootial** [comment]
@mktcode @samuveth  can you help merge this pr? thx

**Comment by samuveth** [comment]
> @mktcode @samuveth  can you help merge this pr? thx

@bonustrack wants to add a review

**Comment by samuveth** [comment]
This is delayed until we have a design.

**Comment by mktcode** [comment]
We should go for something similar than this: 
image

I'll add a suggestion shortly.

**Review by mktcode — APPROVED** [review]

**Comment by samuveth** [comment]
> Btw... I also removed the back button
> image
> since the space name can link to the space page and other than that there's no reason for it. We don't need to replicated browser functionality.

The back button is used when going back to the timeline.

**Comment by samuveth** [comment]
Yes the second one would be my favorite too. Can we add those change before merging :)

**Comment by samuveth** [comment]
@mktcode added those changes and reduced some code for user names.

File: src/components/Modal/RevokeDelegate.vue [hunk 1]
```diff
@@ -67,7 +61,13 @@ async function handleSubmit() {
     <form @submit.prevent="handleSubmit" class="flex flex-col flex-auto">
       <h4 class="m-4 text-center">
         {{ $t('confirmRemove') }}
-        {{ username }}
+        <User
+          :address="proposal.author"
+          :profile="profiles[proposal.author]"
+          :space="space"
+          :proposal="proposal"
```
Comment (mktcode) [inline src/components/Modal/RevokeDelegate.vue:68]: Kaboom! 
image

Comment (samuveth) [reply src/components/Modal/RevokeDelegate.vue]: Oops, fixed :)

**Comment by mktcode** [comment]
Maybe the proposal status should be on the right, instead of next to the username.

image

The user is of course still open, hopefully. Just the proposal is closed. 🤪

**Comment by samuveth** [comment]
> https://bafybeib5eugpexf4yi62s7n6t4eu64vqvzhm2achhuncom4dkgjixzdr3i.on.fleek.co/#/ens.eth
> 
> Doesn't load the ens name. Only after visiting the proposal and going back.
> 
> image
> 
> (Just realized that this was already the case before, but good moment to fix it.)

Yep should be fixed here https://github.com/snapshot-labs/snapshot/pull/1702

**Comment by samuveth** [comment]
@mktcode fixed :)

**Pull request merged**